### PR TITLE
refactor css

### LINKED ISSUE #3: Refactor CSS files
The `public/style.css` file of the landing page should be broken up into multiple small css files for better code management. The splitting should ideally be on the basis of sections in the html file.

Tasks:
- [x] Create multiple css files in `public/css` dir.
- [ ] split the code of `public/style.css` into those multiple css files in `public/css` dir.
- [ ] link those css files to the `public/index.html` file.

### PR DESCRIPTION
**Pull request by snigdhamistry**
fixes #3

### CODE REVIEW

File: public/css/nav.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    background-color: rgb(1, 1, 23);
+    color: burlywood;
+}
+
```
Comment (itskdhere) [inline public/css/nav.css:10]:
```suggestion

```

**Comment by itskdhere** [comment]
LGTM

**Comment by itskdhere** [comment]
Thank you @snigdhamistry 🙏

**Comment by snigdhamistry** [comment]
😎😎😎

**Pull request merged**